common/test/acceptance/tests/discussion/test_discussion.py

```diff
@@ -218,6 +218,51 @@ def test_page_accessibility(self):
 
 
 @attr('shard_2')
+class DiscussionNavigationTest(BaseDiscussionTestCase):
+    """
+    Tests for breadcrumbs navigation in the Discussions page nav bar
+    """
+
+    def setUp(self):
+        super(DiscussionNavigationTest, self).setUp()
+        AutoAuthPage(self.browser, course_id=self.course_id).visit()
+
+    def test_breadcrumbs_all_threads(self):
```

Comment:
Nit: Could you rename this to be more accurate or split the test case out into multiple tests, not just testing the 'all threads' state?